runtime/Cargo.toml

```diff
@@ -51,6 +51,10 @@ pallet-grandpa = { default-features = false, git = "https://github.com/paritytec
 pallet-ethereum = { default-features = false, git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
 fp-rpc = { default-features = false, git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
 
+pallet-democracy = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "master" }
+pallet-scheduler = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "master" }
+pallet-collective = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "master" }
```

Comment:
we're not adding collective in this PR right?

Reply:
I was removed from the PR, so yes we can remove the dependency.